CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 ## Unreleased
 
 * ignore click in info-messages from deleted in-chat apps
+* data saving: do not send messages to the server if user is the only member of the chat in single-device usage
+* protect metadata: encrypt message's sent date
```

Comment:
@link2xt did I got it correctly that now the Date header is protected/encrypted?